lexer.go

```diff
@@ -110,17 +111,28 @@ loop:
 		case '=', ':', '/':
 			if t.qualifier != qErr {
 				t.mechanism = tokenTypeFromString(l.input[l.start : cursor-size])
+				//t.key = l.input[l.start : cursor-size]
```

Comment:
Is this line commented out with a purpose? Should it be deleted?

Reply:
Nope, it is leftover. Removed. Thank you